### PR TITLE
feat(framework:skip) Add get_fab_metadata function

### PR DESCRIPTION
**Pull request by charlesbvll**
### Description

We have no function to easily obtain the `fab_id` and `fab_version` from a FAB file.

### Related issues/PRs

N/A

### Explanation

Introduce a `get_fab_metadata` function that returns a `fab_version`, `fab_id` tuple given a FAB file as a path or as `bytes`.

### Checklist

- [x] Implement proposed change
- [x] Make CI checks pass
- [x] Ping maintainers on [Slack](https://flower.ai/join-slack/) (channel `#contributions`)

### Any other comments?

N/A

### CODE REVIEW

File: src/py/flwr/cli/config_utils.py [hunk 1]
```diff
@@ -14,14 +14,56 @@
...
-from typing import Any, Dict, List, Optional, Tuple
+from typing import IO, Any, Dict, List, Optional, Tuple, Union
 
 import tomli
 
 from flwr.common import object_ref
 
 
+def get_fab_metadata(fab_file: Union[Path, bytes]) -> Tuple[str, str]:
+    """Start a Flower client node which connects to a Flower server.
```
Comment (danieljanes) [inline src/py/flwr/cli/config_utils.py:28]: C&P?

File: src/py/flwr/cli/config_utils.py [hunk 2]
```diff
@@ -14,14 +14,56 @@
...
+    Parameters
+    ----------
+    fab_file : Union[Path, bytes]
+        The Flower App bundle file to validate and extract the metadata from.
+        It can either be a path to the file or the file itself as bytes.
+
+    Returns
+    -------
+    Tuple[str, str]
+        The `fab_version` and `fab_id` of the given Flower App bundle.
```
Comment (danieljanes) [inline src/py/flwr/cli/config_utils.py:39]:
```suggestion
        The `fab_version` and `fab_id` of the given Flower App Bundle.
```

**Review by danieljanes — COMMENTED** [review]
Shouldn't we also start to use the `get_fab_metadata` function in some part of the codebase?

**Comment by charlesbvll** [comment]
> Shouldn't we also start to use the `get_fab_metadata` function in some part of the codebase?

@danieljanes I don't think we currently need it

**Review by danieljanes — APPROVED** [review]

**Pull request merged**